package.json

```
@@ -24,5 +25,9 @@
   "homepage": "https://github.com/carrigan-community/handbook#readme",
   "dependencies": {
     "docsify-cli": "4.4.4"
+  },
+  "devDependencies": {
+    "markdownlint": "^0.28.1",
+    "markdownlint-cli": "^0.33.0"
   }
```

```suggestion
  "devDependencies": {
    "markdownlint": "0.28.1",
    "markdownlint-cli": "0.33.0"
  }
```
These will need to be pinned and the `package-lock.json` will need to be synced (`npm i` + commit and push)

---

It was just fixed.